### PR TITLE
jftui: init at 0.2.2

### PR DESCRIPTION
**Pull request by nyanloutre**
###### Motivation for this change

CLI frontend for Jellyfin. It uses MPV for video playback.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/jftui/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    (fetchpatch {
+      name = "curl-capability-check-fix";
+      url = "https://github.com/Aanok/jftui/commit/d63996b8bc0d2ac4b04c5de4169bc7f8ec9b2a30.patch";
+      sha256 = "1d595mkzgx3carq2cykxpvmf5klgdlyaq94fk9wj8812yswqlsr7";
+    })
+  ];
+
+  buildInputs = [
+    clang
+    pkg-config
```
Comment (jonringer) [inline pkgs/applications/video/jftui/default.nix]: pkg-config should be in nativeBuildInptus

File: pkgs/applications/video/jftui/default.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  pkg-config, curl, mpv, yajl }:
+
+stdenv.mkDerivation rec {
+  pname = "jftui";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "Aanok";
+    repo = pname;
+    rev = "2a65a867c6d4fca4b8b05bf1aedda06d9a3a836a";
```
Comment (jonringer) [inline pkgs/applications/video/jftui/default.nix]:
```suggestion
    rev = "v${version}";
```

**Comment by jonringer** [comment]
please squash commits

**Comment by nyanloutre** [comment]
@jonafato done :)

**Review by jonringer — APPROVED** [review]
LGTM
commit LGTM

```
[2 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/80276
1 package built:
jftui
```

**Pull request merged**